controllers/argocd/statefulset.go

```diff
@@ -85,6 +85,11 @@ func newStatefulSetWithSuffix(suffix string, component string, cr *argoprojv1a1.
 
 func (r *ReconcileArgoCD) reconcileRedisStatefulSet(cr *argoprojv1a1.ArgoCD) error {
 	ss := newStatefulSetWithSuffix("redis-ha-server", "redis", cr)
+	nodePlacementSs := newStatefulSetWithSuffix("redis-ha-server", "redis", cr)
```

Comment:
same! the new stateful set that is being created will not have NodeSlector and Tolerations spec.